tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,185 @@
+
+2. To deny Jack from carrying out the `createCar` transaction, we need to tweak the generated smart contract slightly. We can replace the function with the following: 
+<details open="true">
+<summary> createCar() </summary>
+
+```
+public async createCar(ctx: Context, carId: string, value: string): Promise<void> {
+    const identity: ClientIdentity = ctx.clientIdentity;
+    // Check if the identity has the 'manufacturer' attribute set to 'true'
+    const checkAttr: boolean = identity.assertAttributeValue('manufacturer', 'true');
+    if (checkAttr) {
+        const exists = await this.carExists(ctx, carId);
+        if (exists) {
+            throw new Error(`The car ${carId} already exists`);
+        }
+        const car = new Car();
+        car.value = value;
+        const buffer = Buffer.from(JSON.stringify(car));
+        await ctx.stub.putState(carId, buffer);
+        } else {
+            throw new Error('You must be a manufacturer to carry out this transaction!');
+        }
+}
+```
+>Note: You will need to import `ClientIdentity` by adding the following line at the top of the file (along with the other imports):
+```
+import { ClientIdentity } from 'fabric-shim';
+```
+</details>
+
+***Save*** the changes.
+
+The above simply checks if the identity trying to access the `createCar()` function has the attribute `manufacturer` set to `true`. If it does, allow the transaction to be executed. If the identity does not have the attribute `manufacturer` (or it isn't set to true), do not let the identity carry out the transaction and show an error message.
+
+> Alternatively, you could use the <a href='https://fabric-shim.github.io/release-1.4/fabric-contract-api.Contract.html#beforeTransaction__anchor'> beforeTransaction()</a> method. This method could be used to seperate your business logic from your permissions.
+
+3. To ensure that the above changes are included within our contract, we must `package`, `install` and then `instantiate` the contract.
+</details>
+
+<details>
+<summary><b>4. Package the smart contract</b></summary>
+
+1. In the left sidebar, click on the __IBM Blockchain Platform__ icon.
+
+2. Mouse-over the `SMART CONTRACTS` panel, click the `...` menu, and select `Package Open Project` from the dropdown.
+ 
+> Command Palette alternative: `Package Open Project`
+
+3. You should see a new package on the list, `carContract@0.0.1`, if everything went well.
+
+</details>
+
+<details>
+<summary><b>5. Install the smart contract</b></summary>
+
+To do this...
+
+1. In the `Fabric Environments` panel, look for `+ Install` (it's under Smart Contracts > Installed) and click it.
+
+2. You'll be asked to choose a package to install. Pick `carContract@0.0.1`.
+
+You should see `carContract@0.0.1` appear under the Smart Contracts > Installed list.
+
+> Command Palette alternative: `Install Smart Contract`
+
+That's it - job done! Next up, we'll instantiate the smart contract...
```

Comment:
I don't think you need this either, it makes it sound like you've finished a big task